tests/util/properties.css

```diff
@@ -0,0 +1,17 @@
+@property --any-inherited {
+	syntax: "*";
+	inherits: true;
+	initial-value: 0;
+}
+
+@property --any-non-inherited {
+	syntax: "*";
+	inherits: false;
+	initial-value: 0;
+}
+
+@property --color {
```

Comment:
What is this used for? If it's used for testing typed properties, I would stick to numbers and just use `<number>`. Colors are a whole different beast, and we don't want potential color-related bugs to cloud the test results here. Remember **unit tests only test one thing, if they test multiple things then you don't know how to interpret failures.**